content/ja/handbook/engineering/performance.md

```
@@ -0,0 +1,361 @@
+upstream_path: "/handbook/engineering/performance/"
+upstream_sha: "0e6f01390a34aeb6706ace17d8d3c50e74e82d0d"
+translated_at: "2026-04-29T05:50:35Z"
+translator: claude
+stale: false
+---
+
+## パフォーマンスのファセット
+
+パフォーマンスを 3 つのファセットに分類します。
+
+1. [バックエンド](#backend-performance)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)